osrm.py

```diff
@@ -62,18 +63,25 @@ class OSRMClientException(OSRMException):
     pass
 
 
+def _check_pairs(items):
```

Comment:
Maybe it will be good to check here that p[0] in [0 .. 360] and p[1] in [0 .. 180]